CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org).
 
+## Supported Release 7.0.0
+### Summary
+
+This release includes a change that updates Hiera from version 4 to version 5. This will result in backwards incompatible changes therefore the lower puppet version boundary has been changed to 4.9.4.
```

Comment:
Maybe more like "Since Hiera 5 only works with Puppet 4.9.4 and above, we've bumped the Puppet requirement for the module accordingly"?